data-prepper-api/src/main/java/org/opensearch/dataprepper/model/event/DefaultEventHandle.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.model.event;
+
+import org.opensearch.dataprepper.model.acknowledgements.AcknowledgementSet;
+import java.lang.ref.WeakReference;
+
+import java.time.Instant;
+import java.io.Serializable;
+
+public class DefaultEventHandle implements EventHandle, Serializable {
```

Comment:
Can we make this package protected? I'd really like to avoid this getting used as it will be hard to clean-up in the future.

Reply:
By "package protected"  you mean make this as 
`protected class DefaultEventHandle implements EventHandle, Serializable`?